UnicornRender/system/source/profiler/KeyProfiler.cpp

```diff
@@ -20,7 +20,7 @@ namespace system
 KeyProfiler::KeyProfiler(Manager& manager)
     : m_systemManager( manager )
 {
-    LOG_INFO("KeyProfiler created");
+    spdlog::get(utility::Logger::LoggerName())->info("KeyProfiler created");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.